### PR TITLE
Support list/tuple inputs for special tokens in MultiSegmentPacker layer

### PR DESCRIPTION
**Pull request by abheesht17**

### CODE REVIEW

File: keras_nlp/layers/multi_segment_packer.py [hunk 1]
```diff
@@ -187,22 +217,40 @@ def _combine_inputs(self, segments):
         dtype = segments[0].dtype
         batch_size = segments[0].nrows()
         start_value = tf.convert_to_tensor(self.start_value, dtype=dtype)
+        sep_value = tf.convert_to_tensor(self.sep_value, dtype=dtype)
         end_value = tf.convert_to_tensor(self.end_value, dtype=dtype)
 
-        start_column = tf.fill((batch_size, 1), start_value)
-        end_column = tf.fill((batch_size, 1), end_value)
-        ones_column = tf.ones_like(start_column, dtype=tf.int32)
+        start_values_tensor = tf.repeat(
```
Comment (mattdangerw) [inline keras_nlp/layers/multi_segment_packer.py:223]: these names are a little confusing `start_value` is already a tensor. should we co back to `_column` naming?

`start_column`, `end_column`, `sep_column`?

File: keras_nlp/layers/multi_segment_packer_test.py [hunk 2]
```diff
@@ -147,6 +147,52 @@ def test_pad_batched_inputs(self):
...
+            pad_value="[PAD]",
+            truncate="round_robin",
+        )
+        output = packer([seq1, seq2])
+        self.assertAllEqual(
+            output,
+            (
+                [
+                    [
+                        "[CLS]",
```
Comment (mattdangerw) [inline keras_nlp/layers/multi_segment_packer_test.py:167]: try to come up with a slightly shorter test case that will format the lists to one line

**Review by mattdangerw — CHANGES_REQUESTED** [review]
looks great! just minor comments

File: keras_nlp/layers/multi_segment_packer_test.py (hunk 2 above)
Comment (abheesht17) [reply keras_nlp/layers/multi_segment_packer_test.py:167]: Whoops!

**Review by mattdangerw — APPROVED** [review]
Very nice! Thank you.

**Pull request merged**